Projects/UOContent/Misc/BuffIcons.cs

```diff
@@ -228,57 +228,190 @@ public enum BuffIcon : short
     {
         DismountPrevention = 0x3E9,
         NoRearm = 0x3EA,
-
-        // Currently, no 0x3EB or 0x3EC
-        NightSight = 0x3ED, // *
+        //Currently, no 0x3EB or 0x3EC
+        NightSight = 0x3ED,	//*
```

Comment:
I forget, what does the `*` comment mean?

Reply:
i dont know, its from runUO era. I think its checked bufficon.